design/api.go

```diff
@@ -17,13 +17,19 @@ var _ = API("fuseml", func() {
 		Services("runnable", "openapi")
 
 		// List the Hosts and their transport URLs.
-		Host("localhost", func() {
-			Description("Run the service on localhost.")
+		Host("dev", func() {
+			Description("Run the service listening on localhost.")
 			// Transport specific URLs, supported schemes are:
 			// 'http', 'https', 'grpc' and 'grpcs' with the respective default
 			// ports: 80, 443, 8080, 8443.
 			URI("http://localhost:8000")
 			URI("grpc://localhost:8080")
 		})
+
+		Host("prod", func() {
```

Comment:
Same as above, instead of using an anonymous function, just use a named function with the right parameter.

Reply:
This comes from the Goa DSL, I tried using a named function and it did not work as it expects a `DSL function`.

Reply:
I can dig there with you if you don't mind.